src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const title = movie.title.toLowerCase();
+    const description = movie.description.toLowerCase();
+    const queryLowerCase = query.toLowerCase().trim();
+
+    if (title.includes(queryLowerCase)) {
+      return true;
+    }
+
+    if (description.includes(queryLowerCase)) {
+      return true;
+    }
+
+    return false;
+  });
```

Comment:
```suggestion
    return title.includes(queryLowerCase) || description.includes(queryLowerCase);
  });
```
You can simplify that ;)